moveit_setup_assistant/templates/moveit_config_pkg_template/launch/gazebo.launch

```diff
@@ -1,31 +1,43 @@
 <?xml version="1.0"?>
 <launch>
+
+  <!-- Gazebo specific options -->
+  <arg name="world_name" default="worlds/empty.world"/>
   <arg name="paused" default="false"/>
   <arg name="gazebo_gui" default="true"/>
-  <arg name="initial_joint_positions" doc="Initial joint configuration of the robot"
-       default="[GAZEBO_INITIAL_JOINTS]"/>
 
-  <!-- startup simulated world -->
-  <include file="$(find gazebo_ros)/launch/empty_world.launch">
-    <arg name="world_name" default="worlds/empty.world"/>
+  <!-- Robot spawn pose -->
+  <arg name="x" default="0"/>
+  <arg name="y" default="0"/>
+  <arg name="z" default="0"/>
+  <arg name="R" default="0"/>
+  <arg name="P" default="0"/>
+  <arg name="Y" default="0"/>
+
+  <!-- Spawn options -->
+  <arg name="unpause" doc="Unpause only after loading robot model argument" value="$(eval '' if arg('paused') else '-unpause')"/>
+  <arg name="world_pose" doc="Robot spawn pose argument" default="-x $(arg x) -y $(arg y) -z $(arg z) -R $(arg R) -P $(arg P) -Y $(arg Y)"/>
+  <arg name="initial_joint_positions" doc="Initial joint configuration of the robot argument" default="[GAZEBO_INITIAL_JOINTS]"/>
+
+  <!-- Start Gazebo -->
+  <include file="$(find gazebo_ros)/launch/empty_world.launch" pass_all_args="true">
+    <arg name="world_name" value="$(arg world_name)"/>
     <arg name="paused" value="true"/>
```

Comment:
```suggestion
    <arg name="paused" value="$(arg paused)"/>
```

Reply:
Actually Gazebo always has to start paused. Otherwise the robot cannot be spawned with the initial_joint_positions.

Reply:
Sorry. I'm afraid I misinterpreted this code the first time. While you want Gazebo itself to be started in _paused_ mode _initially_, later - after robot controller initialization - you might want to unpause it, right?
From your recent commit, I recognized that the pause argument was used to control this latter unpausing. So, sorry for the noise but please revert that part (the one related to `pause`)